manifest-build-tools/application/update_manifest.py

```diff
@@ -0,0 +1,442 @@
+#!/usr/bin/env python
+
+# Copyright 2015, EMC, Inc.
+
+"""
+Usage:
+./on-tools/manifest-build-tools/HWIMO-BUILD on-tools/manifest-build-tools/application/update_manifest.py \
+--repo "$url" \
+--branch "$branch" \
+--commit "$id" \
+--manifest_download_url https://dl.bintray.com/rackhd-mirror/binary/manifest/ \
+--git-credential https://github.com,GITHUB_CREDS \
```

Comment:
The parameter git_credential is optional for the class ```RepoOperator``` and ```Manifest```.
I think the parameter should be optional to the script update_manifest.